src/pybricks/_common.py

```diff
@@ -335,7 +335,7 @@ def __init__(
                 Choose ``False`` to keep the
                 current value, so your program knows where it left off last
                 time.
-        """
+        """   # noqa: E501
```

Comment:
Instead of repeating this multiple times, let's just add `E501` to the `ignore` list in `setup.cfg` and remove all of the `noqa: E501`s.

Reply:
done! -- yes, this is consistent with your use of Black mentioned above.